justfile

```diff
@@ -0,0 +1,101 @@
+# DO NOT EDIT BY HAND: is automatically re-generated by make.py
```

Comment:
Same as the comment on `Makefile`: if a line like this has to exist, it seems like a problem.

Reply:
We could just `.gitignore` them, then emit them whenever you run `make.py`.  So anyone who wants to convenience of `make` will get it right after they run `make.py` for the first time.